app/serializers/api/v1/orders_package_serializer.rb

```diff
@@ -3,7 +3,7 @@ class OrdersPackageSerializer < ApplicationSerializer
     embed :ids, include: true
     attributes :id, :package_id, :order_id, :state, :quantity, :dispatched_quantity, :sent_on, :designation_id, :item_id, :created_at, :allowed_actions
 
-    has_one :package, serializer: PackageSerializer
+    has_one :package, serializer: PackageSerializer, root: :item
```

Comment:
This might cause problems elsewhere. In stock we use `item` as root. But in browse it's used as `packages`.

I think it'll break on browse. Have you tested browse after this change ? 

cc- @namrataukirde

Reply:
@abhinav098 @abulsayyad123 @namrataukirde 

I suggest we create a JIRA ticket to refactor Stock's models. Meaning `item` into `package`, and `designation` into `order` to restore consistency. After which all this drama can go away

Reply:
Yes, I am in 😄